src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions_Parsing.cs

```diff
@@ -56,6 +56,7 @@ public static CodeStyleOption2<AddImportPlacement> ParseUsingDirectivesPlacement
             {
                 "inside_namespace" => new CodeStyleOption2<AddImportPlacement>(AddImportPlacement.InsideNamespace, notification),
                 "outside_namespace" => new CodeStyleOption2<AddImportPlacement>(AddImportPlacement.OutsideNamespace, notification),
+                "outside_namespace_ignoring_aliases" => new CodeStyleOption2<AddImportPlacement>(AddImportPlacement.OutsideNamespaceIgnoringAliases, notification),
```

Comment:
intentionally using hte word `ignoring` (as opposed to `excluding` or `except`) as i don't want to convey taht aliases must then be *required* to be inside the namespace.  Instead, this is trying to say that aliases are simply ignored and can appear either outside or inside the namespace (for example, roslyn follows this model).